enhancements/dns/coredns-egressfirewall-integration.md

```diff
@@ -0,0 +1,761 @@
+---
+title: coredns-egressfirewall-integration
+authors:
+  - '@arkadeepsen'
+reviewers:
+  - '@Miciah'
+  - '@danwinship'
+  - '@JoelSpeed'
+  - '@TrilokGeer'
+  - '@jerpeter1'
```

Comment:
It can be helpful to explain the expertise each reviewer is bringing, what part of this do you want their input on